sections/06_Transactions.md

```diff
@@ -0,0 +1 @@
+# Interacting with NEO network: transactions, RPC and P2P protocols
```

Comment:
Should this part be included in the section "Network"?

Reply:
Done.